pkgs/development/libraries/libpsl/default.nix

```diff
@@ -31,7 +31,7 @@ stdenv.mkDerivation rec {
     lzip
     pkgconfig
     python3
-    valgrind
+    (stdenv.lib.optionalString (!stdenv.isDarwin) valgrind)
     libxslt
   ];
```

Comment:
```suggestion
    libxslt
  ] ++ stdenv.lib.optionals (!stdenv.isDarwin) [ valgrind ];
```
besides that, LGTM

Reply:
Thats is what I did [first](https://github.com/NixOS/nixpkgs/compare/2c28ff2f00730b8f9a3c1547ad9830287de57191..c635f8f156886ca712ec89c431a50ee9bdea4bda) but it causes unnecessary rebuilds on linux.

Reply:
oh, interesting, I'm not sure why it should cause rebuilds on Linux

Reply:
The order of build inputs is preserved into the build process itself.

Reply:
TIL thanks!